src/scss/base/_nav.scss

```diff
@@ -1,62 +1,96 @@
-.nav-items {
-    float: right;
-    padding: 0;
-    margin-right: 150px;
+// phần 3 cục
+.clump {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 100px 80px;
 
-    .nav-item {
-        display: inline-block;
-    }
+  .area {
+    background-color: #00cdcd;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 300px;
+    width: 400px;
+    margin: 20px;
+    text-align: center;
+    padding: 40px;
+    position: relative;
+    border: 3px solid transparent;
+  }
+
+  .title {
+    color: #ffffff;
+    font-size: 1.5rem;
+    padding: 20px;
+  }
+
+  .note {
+    color: #ffffff;
+    font-size: 18px;
+  }
 
-        .nav-link {
-            padding: 20px;
-            color: #ffffff;
-            text-decoration: none; 
-        }    
+  .sub-area {
+    height: 120px;
+    width: 120px;
+    background-color: #ffffff;
+    border-radius: 50%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    position: absolute;
+    top: 5%;
+    left: 50%;
+    transform: translate(-50%);
+
+    .icon {
+      margin: auto;
+      color: #00cdcd;
+    }
+  }
 }
-.nav-link:hover {
-    border-bottom: 2px solid #ffffff;
+
+.area:hover {
+  border-bottom: 3px solid #f2ce5f;
+
+  .fas {
+    color: #f2ce5f;
+  }
 }
+// phần phone
 
-.dropdown {
-    position: relative;
+.main-phone {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background-image: url(../images/gray-bg.jpg);
+  background-size: cover;
+  background-position: center;
+  width: 100%;
```

Comment:
tại sao cần width: 100%?

Reply:
e ko bt ảnh nó to như thế nên e fix cho chắc ^_^